contracts/deploy/mainnet/097_withdraw_queue.js

```diff
@@ -0,0 +1,64 @@
+const addresses = require("../../utils/addresses");
+const { deploymentWithGovernanceProposal } = require("../../utils/deploy");
+
+module.exports = deploymentWithGovernanceProposal(
+  {
+    deployName: "097_withdraw_queue",
+    forceDeploy: false,
+    //forceSkip: true,
+    reduceQueueTime: true,
+    deployerIsProposer: false,
+    // proposalId:
+  },
+  async ({ deployWithConfirmation }) => {
+    // Deployer Actions
+    // ----------------
+
+    // 1. Deploy new OETH Vault Core and Admin implementations
+    // Need to override the storage safety check as we are repacking the
+    // internal assets mapping to just use 1 storage slot
+    const dVaultCore = await deployWithConfirmation(
+      "OETHVaultCore",
+      [addresses.mainnet.WETH],
+      null,
+      true
+    );
+    const dVaultAdmin = await deployWithConfirmation(
+      "OETHVaultAdmin",
+      [],
+      null,
+      true
+    );
+
+    // 2. Connect to the OETH Vault as its governor via the proxy
+    const cVaultProxy = await ethers.getContract("OETHVaultProxy");
+    const cVault = await ethers.getContractAt("OETHVault", cVaultProxy.address);
+    const cDripperProxy = await ethers.getContract("DripperProxy");
+
+    // Governance Actions
+    // ----------------
+    return {
+      name: "Upgrade OETH Vault to support queued withdrawals",
+      actions: [
+        // 1. Upgrade the OETH Vault proxy to the new core vault implementation
+        {
+          contract: cVaultProxy,
+          signature: "upgradeTo(address)",
+          args: [dVaultCore.address],
+        },
+        // 2. set OETH Vault proxy to the new admin vault implementation
+        {
+          contract: cVault,
+          signature: "setAdminImpl(address)",
+          args: [dVaultAdmin.address],
+        },
+        // 3. Set the Dripper contract
```

Comment:
Did I miss something? This PR shouldn't have to mess with the dripper? And doesn't make any contract changes to the dripper?

Reply:
Nevermind!

Reply:
This PR includes the change to the Vault to call `collect` on the `Dripper` on mints to prevent yield theft.